puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/release-nightly-build-rpm.groovy

```diff
@@ -0,0 +1,38 @@
+def releaserArgs = []
+
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage("Setup Environment") {
+            steps {
+                git(url: 'https://github.com/theforeman/foreman-packaging/', branch: 'rpm/develop')
+                setup_obal()
+            }
+        }
+
+        stage("Obal Nightly Build RPM") {
+            steps {
+                script {
+                    if (env.jenkins_job) {
+                        releaserArgs << "--arg jenkins_job=${env.jenkins_job}"
+                    }
+                    if (env.jenkins_job_id) {
```

Comment:
You could argue that if either of these parameters are empty it's actually a misconfiguration and you could fail on it before git clone.

Reply:
The fact that the `jenkins_job` parameter defaults to nothing makes me lean towards no failing the job if the parameter is left empty.

Reply:
Hmm, you're probably right since the manifest includes the job name already it may not be needed.